nldi-liquibase/src/main/resources/liquibase/characteristic_data/indexes.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+
+--changeset drsteini:create_index.characteristic_data.plusflowline_vaa_np21_ut
+--preconditions onFail:MARK_RAN onError:HALT
+--precondition-sql-check expectedResult:0 select count(*) from pg_class c join pg_namespace n on n.oid = c.relnamespace where n.nspname = 'characteristic_data' and c.relname = 'plusflowlinevaa_np21_ut' and c.relkind = 'i'
```

Comment:
Is this precondition checking for the existence of the index? If so, is it a better approach to place this query in the precondition, or to eliminate the precondition in favor of a `CREATE INDEX IF NOT EXISTS` changeset entry?